en/docs/develop/customizing-my-account-ui.md

```diff
@@ -0,0 +1,237 @@
+# Customizing the My Account UI
+
+From [WSO2 Identity Server 5.10.0](https://wso2.com/identity-and-access-management/) onwards, a new **My Account** has
+been introduced which is the successor to the dashboard. It also adheres to the UI extensions scheme
+ introduced with WSO2 Identity Server 5.9.0 to use a centralized theming for all the front-end portals. 
+
+!!! note 
+    The **My Account** application has been renamed as **My Account** from this release onwards.
+
+This section explains how we can customize the theming of **My Account**. 
+
+## Changing the default theme to dark mode
+
+!!! info
+    A customized version of the [default theme](https://github
+    .com/Semantic-Org/Semantic-UI-LESS/tree/master/themes/default) in the Semantic UI LESS package has been used to
+    achieve the look and feel of the **My Account**.
+
+    ![default-my-account-theme](../assets/img/develop/default-my-account-theme.png)
+
+    For information on the Semantic UI theming, see [Semantic UI documentation](https://semantic-ui.com/usage/theming.html).
+
+!!! note "Before you begin"
```

Comment:
since there is already an info macro right above this, wdyt about making this normal text under a subheading "Before you begin" with a line separator after it? Might look neater on the page